### PR TITLE
update react-native-status-keycard

### LINKED ISSUE #14815: Keycard NFC tag remove Issue: Java Security Exception
## Problem 
Pairing the Keycard on Android 12 in Status.im (`Device 1`) causes problems.
Since some time Android 12 seems to have problem with the NFC connectivity when pairing and connecting the keycard in Status as well as signing transactions.

#### Expected behavior
When pairing a fresh keycard with the newest applet my Status App in Android 12, the app should go through the pairing process, requesting the pairing password and give access to the wallet.

#### Actual behavior
The following Java Runtime Error occurs when pairing a new card or connecting an existing card:

`AndroidRuntime: java.lang.SecurityException: Permission Denial: Tag ( ID: XX XX XX XX XX XX XX ) is out of date`

So this leads me to the conclusion that something happens during the NFC Rediscover routine that is not suppose to happen (maybe due to an uncatched exception when a Nfc reconnect happens in Android 12 when the card is not placed correctly on the sensor, which happens a lot during discovery).

### Reproduction

1) Open Status
2) Pair an existing keycard or connect an already paired keycard to the status app on Android 13. 
3.) When scanning the NFC Card, the following Java Exception Occurs:

 `AndroidRuntime: java.lang.SecurityException: Permission Denial: Tag ( ID: XX XX XX XX XX XX XX ) is out of date.` 

The app should crash after that exception.

4.) Other errors (APDU failures on `Device 2`) are also observed if connecting the card to an Android 11 after step 1 & 2.

`AndroidRuntime: java.lang.IllegalArgumentException: APDU response must be at least 2 bytes`

The pairing and connecting of the Keycard in Android 10 however still works.

### Additional Information
`Device 1:`
- Status version: 1.20
- Operating System: Android 12 (CalyxOS  4.5.0)
- Device: Fairphone 4 (PF4)

`Device 2:`
- Status version: 1.20
- Operating System: Android 11
- Device: Samsung Galaxy A50

After some investigation this seems to be a known issue at AOSP:

https://issuetracker.google.com/issues/238257723

It seems that this SecurityException was introduced in AOSP commit [113f3a4]
It is thrown when an NFC tag is removed.

The relevant Diff is given here:
https://android.googlesource.com/platform/frameworks/base/+/113f3a4a1d9171db7456aa9ae8b0a2b50545c326%5E%21/

**A history of the relevant code and the original Android 12 upstream issue at is attached below:** - NFC Tag Routines in the Android Nfc Service:

Methods: 
`void setTagUpToDate(long cookie)`
`boolean isTagUpToDate(long cookie)`

The return values from this functions should be checked as well as Catched Exceptions on tag removal.

https://android.googlesource.com/platform/packages/apps/Nfc/+/master/src/com/android/nfc/NfcService.java#2124
https://android.googlesource.com/platform/frameworks/base/+/master/core/java/android/nfc/Tag.java#380

An example fix for reference:
https://github.com/open-keychain/open-keychain/pull/2804

### PR DESCRIPTION
**Pull request by bitgamma**
Fixes #14815

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -69,7 +69,7 @@
     "react-native-safe-area-context": "^2.0.0",
     "react-native-shake": "^3.3.1",
     "react-native-share": "^7.0.1",
-    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#refs/tags/v2.5.38",
+    "react-native-status-keycard": "git+https://github.com/status-im/react-native-status-keycard.git#fix-security-exception",
```
Comment (jakubgs) [inline package.json]: Please remember to tag it and update this before merging.

**Review by jakubgs — APPROVED** [review]

**Comment by jakubgs** [comment]
You really shouldn't merge stuff before all tests are green dude. If you rebased and ran builds again it would have included the fix from #15167 and it should have worked, but now this commit is in question since it's not all green.

**Comment by bitgamma** [comment]
> You really shouldn't merge stuff before all tests are green dude. If you rebased and ran builds again it would have included the fix from #15167 and it should have worked, but now this commit is in question since it's not all green.

which tests are you talking about exactly? e2e? we have already checked with @churik and @cammellos that it was OK to merge. I also did rebase before merging of course.

**Comment by jakubgs** [comment]
iOS:

image

But really most are marked as still pending:

image

**Comment by bitgamma** [comment]
ah you mean the builds. Yeah sorry about that, didn't pay attention to iOS build status cause all this PR does is bump the version of the Keycard SDK for Android. There is no reasonable way this can cause breaking iOS build. The changes to the react-native-status-keycard module are here https://github.com/status-im/react-native-status-keycard/commit/93dd64754e676172310e6ea7187cc49f2dc013c6, again only bumps the keycard sdk for android.

**Comment by jakubgs** [comment]
Sure, you're right about it being just android-related, but when you're looking through the commits when trying to debug an issue an indicator of failed/pending builds makes it a prime suspect, which is just a red herring in this case, which wastes time. It's not a huge thing, mostly a nitpick, but it does help to have clean commits in `develop` that have been build and tested successfully.

Good job on fixing this anyway. Just a note for the future.

**Pull request merged**